src/app/Library/CrudPanel/Traits/SaveActions.php

```diff
@@ -123,32 +332,94 @@ public function performSaveAction($itemId = null)
                 'success'      => true,
                 'data'         => $this->entry,
                 'redirect_url' => $redirectUrl,
+                'referrer_url' => $referrer_url,
             ];
         }
 
+        if (isset($referrer_url)) {
+            session()->flash('referrer_url_override', $referrer_url);
+        }
+
         return \Redirect::to($redirectUrl);
     }
 
     /**
-     * Get the translated text for the Save button.
+     * Returns the default Backpack save actions for cruds.
      *
-     * @param string $actionValue
+     * @return array
+     */
+    public function setupBackpackDefaultSaveActions()
+    {
+        $defaultSaveActions = [
+            [
+                'name' => 'save_and_back',
+                'permissions' => function ($crud) {
```

Comment:
Since this is a closure now, that always returns true/false, I think we can now make this easier to understand and move away from the "permissions" terminology, that confuses users (because there may also be roles & permissions from the spatie package).

I propose we change ```permissions``` here to ```visible```. Since its returned value is ```true``` or ```false``` this should be super-clear now: 
- _visible ```true``` or visible ```false```;_
- _how's visible determined? let's look at the closure... ah ok, it checks for access to the list operation;_

No mention of "permission", anywhere. 

This should eliminate some more of the confusion around this feature. Looks like when I've renamed "permissions" to "access" in v4 I've missed a spot, in SaveActions. Time now to remove it from here too.

Reply:
The name seemed odd to me but well, I know what it does, it was good for starter. I refactored of course, agree 100% with you.